app/assets/javascripts/layout.js

```diff
@@ -13,15 +13,23 @@
     $('textarea.text', element).summernote();
   }
 
+  // Dropdown does not work on iOS devices as they only support click delegation for
+  // Proposed solution is modified from http://stackoverflow.com/a/22318440
+  // Event delegation for iOS http://www.quirksmode.org/blog/archives/2010/09/click_event_del.html
+  function initializeDropdownEventListener() {
+    $('[data-toggle=dropdown]').each(function() {
+      this.addEventListener('click', function() {}, false);
+    });
+  }
+
   function initializeComponents(element) {
     $('[data-toggle="popover"]', element).popover();
-    // The design of Coursemology is such that tooltips are attached to any
-    // element with a title attribute, but we do not want that for the Facebook
-    // button
+    // Tooltips are attached to elements with a title attribute, except for the Facebook button.
     // See https://github.com/Coursemology/coursemology-theme/pull/5
     $('[title]', element).not('.fb-like *').tooltip();
     $('input.toggle-all[type="checkbox"]', element).checkboxToggleAll();
     $('textarea.code', element).ace();
+    initializeDropdownEventListener();
```

Comment:
I will only really need it if 2 conditions are met: 
 - There are dropdown menus in that page (which is almost every page because of the navbar)
 - If it is loaded on an iOS device.

Should I find another way to do this @allenwq ?